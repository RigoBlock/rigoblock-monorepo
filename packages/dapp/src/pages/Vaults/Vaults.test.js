import { shallow } from 'enzyme'
import React from 'react'
import toJson from 'enzyme-to-json'

describe('Vaults page', () => {
  const apiMock = {
    contract: {
      DragoRegistry: {
        address: '0xf7cbb0849d4a8ec5ab4650030fa776c00eb52la4',
        createAndValidate: jest.fn()
      }
    },
    web3: {}
  }
  let Vaults

  beforeEach(() => {
    jest.resetModules()
    jest.doMock('../../api', () => apiMock)

    Vaults = require('./Vaults').default
  })
  it('renders correctly', () => {
    expect(toJson(shallow(<Vaults />))).toMatchSnapshot()
  })
})
