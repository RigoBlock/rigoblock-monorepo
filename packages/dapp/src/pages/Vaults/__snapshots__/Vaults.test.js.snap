// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vaults page renders correctly 1`] = `
<BaseTemplate>
  <h1>
    Vaults
  </h1>
  <Connect(vaultSelect) />
</BaseTemplate>
`;
